frontend/src/metabase/query_builder/components/notebook/steps/JoinStep/JoinComplete/JoinComplete.tsx

```diff
@@ -0,0 +1,186 @@
+import { useMemo, useState } from "react";
```

Comment:
User needs to click the text (name of table) to edit RHS table despite `cursor: pointer` shown for the entire button:

https://github.com/metabase/metabase/assets/6830683/b6f695a9-234d-4a6a-ad36-ec54c219098a

Reply:
Good catch, I'll fix that

Reply:
It's still there, I get it very often.

Reply:
@kamilmielnik fixed. I did it in the same way as it is in DataStep